pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentSubscription.java

```diff
@@ -1096,5 +1096,18 @@ public boolean checkAndUnblockIfStuck() {
         return dispatcher.checkAndUnblockIfStuck();
     }
 
+    @Override
+    public int getNumberOfSameAddressConsumers(final String clientAddress) {
```

Comment:
There is still duplicate code here, can we use default to move it to the interface?
It looks stateless.